### DISCUSSION TITLE
To build a Multi-gpu tensor

### DISCUSSION THREAD

**Discussion by 729009982** [apache/incubator-mxnet#12820] [Q&A]
Hi.

    I want to build a nd.zeros() as the label for cGAN ,but when I key in 
nd.zeros(shape,ctx=[gpu(0),gpu(1),gpu(2),gpu(3)]) 
get:
Invalid context string [gpu(0), gpu(1), gpu(2), gpu(3)]

copy_to(gpus), as_in_context(gpus) and split_and_load(x,gpus) does not work
How can I do ?
Thank you for your time :-D

**Comment by piyushghai** [comment]
@729009982 Thank you for posting your question.
You can also post this question on MXNet discussion forum (https://discuss.mxnet.io), where it will get a wider audience and allow other to learn as well.

@mxnet-label-bot [Question, Python]

**Comment by piyushghai** [comment]
@mxnet-label-bot [NDArray]
